src/pudl/io_managers.py

```diff
@@ -417,15 +446,28 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
                 name.
         """
         table_name = self._get_table_name(context)
-
         # Check if the table_name exists in the self.md object
         _ = self._get_sqlalchemy_table(table_name)
```

Comment:
We shouldn't call `_ = self._get_sqlalchemy_table(table_name)` here because we'll be loading sql views as dataframes which won't have `Table()` objects in `self.md`.

Reply:
Ohhh. Okay. If we reflected & inspected the DB, then we'd find them in **that** metadata, but that information about the table exists only the DB and in the Package, and not in the metadata generated by the Package (since we're creating those tables directly via CREATE VIEW)

Reply:
Correct, though I'm unsure if `sa.Metadata().reflect()` would create Table objects for views. Worth experimenting.